config/src/main/java/org/opennms/integration/api/xml/schema/eventconf/AttributeType.java

```diff
@@ -0,0 +1,89 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2010-2016 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2016 The OpenNMS Group, Inc.
```

Comment:
```suggestion
 * OpenNMS(R) is Copyright (C) 2022 The OpenNMS Group, Inc.
```

Reply:
should it be OpenNMS(R) is Copyright (C) 1999-2022 The OpenNMS Group, Inc.?